src/components/article/components.js

```diff
@@ -1,11 +1,13 @@
-import { Hero } from "@components/hero";
+import { Hero } from "@components/article/hero";
```

Comment:
```suggestion
import { Hero } from "./hero";
```

И можем ли мы сделать так для остальных?